README.md

```diff
@@ -96,6 +98,23 @@ To run the tests locally without docker:
 ./run_tests.sh local
 ```
 
+### Run
+
+You can also run `dcrd` with docker:
```

Comment:
I'd give an example command line of how someone will do this, perhaps at the bottom of this text, e.g. 

```
docker build . -t decred
docker run -v ~/decred_data:/var/lib/dcrd -v ~/decred_config:/etc/dcrd decred 
```

Perhaps it's also useful to explain why this is cool - the fact an average user just needs to git clone and then run these commands and they're running bleeding edge decred from source is a pretty nice feature if they're not otherwise a developer.

Reply:
> I'd give an example command line of how someone will do this, perhaps at the bottom of this text, e.g.

The reason I didn't is because, most users won't need to build it themselves.
I wish that an official image is published to http://hub.docker.com/r/decred

No need to builds it, just need to pull & run.

I don't want to refers to my own build of the image https://hub.docker.com/r/bclermont/decred-dcrd/

If Decred team want to publish the image, we can have doc just point to official image

Reply:
@bclermont Please provide the example @jpz hinted at as it is useful.

Reply:
https://github.com/decred/dcrd/pull/919/commits/a7870fd164cb33789b3e441122ce5a7362f97139

added a docker-compose example